scripts/write-to-disk.js

```diff
@@ -0,0 +1,121 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Write emails to disk. Output is written to ./.mail_output/<email type>.html
```

Comment:
Maybe add a note that `./.mail_output` must exist in advance, it won't be created automatically? Or not, it becomes pretty obvious when you try to run the the script.